### ISSUE TITLE
[AXE-CORE]: ARIA progressbar nodes must have an accessible name

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#21787]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix issue.
1. At message the accessibility specialist when issue is resolved.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement
Accessibility specialists discovered Nightwatch axe checks failing silently during a recent staging review. Multiple axe violations were discovered where ARIA progressbar attributes were not conforming to valid values. Preliminary research indicates this was for a custom version of the [Design System loading indicator](https://design.va.gov/storybook/?path=/story/components-loadingindicator--default). Links to pages below.

Loading indicator = VA Blue Button
![image](https://user-images.githubusercontent.com/55411834/112064899-56233500-8b29-11eb-9b2c-89b705e8e91d.png)

## Acceptance Criteria
- [ ] All instances of the error are fixed
- [ ] Axe-browser plugin or axe-core CLI scans show 0 Section 508, WCAG2A or WCAG2AA violations

## Environment
* https://www.va.gov/erie-health-care/medical-records-office/
* There may be others, but the CI logs identified this single URL

## Steps to Recreate
1. Open one of the listed URLs
1. Open axe browser plugin (assumes already installed in Chrome or Firefox)
1. Confirm the `ARIA progressbar nodes must have an accessible name` is present one or more times

**Comment by kelsonic** [comment]
PR open: https://github.com/department-of-veterans-affairs/vets-website/pull/16956

**Comment by brianalloyd** [comment]
@1Copenut we did some work on this improvement and should be ready for validation when you have bandwidth.  Thanks @kelsonic for the work on this ticket.
